### ISSUE TITLE
Automate airtable data pulls and downstream queries

### ISSUE THREAD

**Issue by rggelles** [georgetown-cset/parat#59]
I'd like to set up an airflow pipeline to automate the process of going from cleaned-up/imported data to final tables.

Depending on difficulty level (I know @za158 has done some work on automating imports from Airtable?) it would be cool to get this pipeline working so that it could even automate the transferring over from Airtable. But even if it doesn't do that, just automating all the python scripts and sql scripts would be super helpful.

**Comment by jmelot** [comment]
Hope you don't mind, updated the title to disambiguate as we/I have another task to automate updates to the webapp

**Comment by rggelles** [comment]
Airtable data pulls have been automated here: #111 

Now on to making a new DAG to automate the downstream queries.

**Comment by rggelles** [comment]
The DAG is nearly complete; awaiting tasks and methods updates to finish off (have to change the linking to the new T&M tables to finish). Also need to finalize the addition of check queries and backups. Will close once this is complete.
